db/migrate/20230406135310_ajoute_bienvenue_sociodemographique_aux_parcours_types.rb

```diff
@@ -0,0 +1,17 @@
+class AjouteBienvenueSociodemographiqueAuxParcoursTypes < ActiveRecord::Migration[7.0]
+  def change
+    ParcoursType.all.each do |parcours_type|
+      situation_bienvenue = Situation.find_by nom_technique: Situation::BIENVENUE
+      questionnaire_socio =
+        Questionnaire.find_by nom_technique: Questionnaire::SOCIODEMOGRAPHIQUE
```

Comment:
Pour éviter de refaire les requête pour chaque parcours type, on peut les sortir de la boucle.
```suggestion
    situation_bienvenue = Situation.find_by nom_technique: Situation::BIENVENUE
    questionnaire_socio =
      Questionnaire.find_by nom_technique: Questionnaire::SOCIODEMOGRAPHIQUE
    ParcoursType.all.each do |parcours_type|
```